doc/guide/build.hpp

```diff
@@ -95,7 +95,8 @@ For Python bindings, the following packages are required:
  - pandas >= 0.15.0
  - pytest-runner
 
-In Ubuntu and Debian, you can get all of these dependencies through apt:
+In Ubuntu(>= 18.04) and Debian all of these dependencies can be 
```

Comment:
Is there also a minimum debian version?

Reply:
I'm not sure about that. Would love input from some Debian user or i can checkout the documentations if no one is available by tomorrow.

Reply:
See https://packages.debian.org/source/buster/armadillo ; it looks like "buster" (10) and "bullseye" (11) (and "sid" (unstable)) have sufficient armadillo.